zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3653,11 +3942,23 @@ components:
         - $ref: "#/components/schemas/SearchQueryRequest"
       description: User task search query request.
       type: object
-    UserTaskSearchQueryResponse:
+    UserTaskSearchQueryResponseNumberKeys:
+      description: User task search query response. (Number keys)
```

Comment:
🔧 The `(Number keys)` documentation is not really clear for users, I believe. We could say something like `with key attributes as numeric values`. Wdyt?

Reply:
I changed to this: `description: Tenant search response item. Key attributes as numeric values.`